doc/api/dgram.markdown

```diff
@@ -155,9 +155,9 @@ this.
 If `multicastInterface` is not specified, the OS will try to drop membership to all valid
 interfaces.
 
-### socket.send(buf, offset, length, port, address[, callback])
+### socket.send(buf, [offset, length,] port, address[, callback])
 
-* `buf` Buffer object or string.  Message to be sent
+* `buf` Buffer object or string, or array of those. Message to be sent
```

Comment:
indeed that would be appropriate

also i would replace the current use of `or` with a comma like

```bash
Buffer object, string, or an array of either.
```